tests/integration/cache.spec.ts

```diff
@@ -115,4 +122,24 @@ describe("Cache", () => {
 
     await expect(cache.get(key, executor, { tags })).resolves.toEqual(struct);
   });
+
+  it("Redis concurrent test", async () => {
```

Comment:
Probably it is needed to extend jest timeout for this test 

https://github.com/Tinkoff/cachalot/runs/5787885398?check_suite_focus=true#step:9:23

Reply:
done